t/t2002-mpibench.t

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+
+test_description='Test openmpi with mpibench.'
+
+PLUGINPATH=${FLUX_BUILD_DIR}/src/shell/plugins/.libs
+MPIBENCH="${FLUX_BUILD_DIR}/t/mpibench/mpibench -i5 -C -e1K -m 1M"
+
+. `dirname $0`/sharness.sh
+
+if ! test_have_prereq LONGTEST; then
+    skip_all='skipping mpibench due to missing LONGTEST prereq'
+    test_done
+fi
+
+test_under_flux 2
+
+cat >tests.dat<<EOT
+60 1 2 Barrier
+60 2 3 Barrier
+60 2 4 Barrier
+60 1 2 Bcast
+60 2 3 Bcast
+60 2 4 Bcast
+60 1 2 Alltoall
+60 2 3 Alltoall
+60 2 4 Alltoall
+60 1 2 Allgather
+60 2 3 Allgather
+60 2 4 Allgather
+60 1 2 Gather
+60 2 3 Gather
+60 2 4 Gather
+60 1 2 Scatter
+60 2 3 Scatter
+60 2 4 Scatter
+60 1 2 Allreduce
+60 2 3 Allreduce
+60 2 4 Allreduce
+60 1 2 Reduce
+60 2 3 Reduce
+60 2 4 Reduce
+EOT
+
+run_mpibench() {
+	local timeout=$1
+	local nnodes=$2
+	local ntasks=$3
+	local op=$4
+	run_timeout $timeout flux mini run -N$nnodes -n$ntasks \
+		-ouserrc=$(pwd)/rc.lua \
+		-ompi=openmpi@5 \
+		${MPIBENCH} ${op}
+}
+
+
+test_expect_success 'create rc.lua script' "
+	cat >rc.lua <<-EOT
+	plugin.load (\"$PLUGINPATH/pmix.so\")
+	shell.env_strip (\"^OMPI_MCA_pmix\", \"^OMPI_MCA_schizo\")
+	shell.setenv (\"OMPI_MCA_btl_tcp_if_include\", \"lo\")
+	EOT
+"
```

Comment:
It occurs to me that another way to handle the setup and load of `pmix.so` for all tests would be to have this rc.lua generated in `t/etc/rc.lua` and then set `FLUX_SHELL_RC_PATH` to this path. Since this works there is no reason to change now, but it may be a way to clean up the testsuite in the future (so you don't have to generate a custom `rc.lua` in each sharness test.

Reply:
Good idea,    I'll give that a whirl and see if I can avoid the replication added in this PR.